src/app/lorax/browser-detection.js

```diff
@@ -0,0 +1,48 @@
+define(['jquery', 'bowser'], function($, bowser) {
+    'use strict';
+
+    function init() {
+        bindEvents();
+
+        var supported = (
+            // IE 10+
+            (bowser.msie && bowser.version >= 10) ||
+
+            // Firefox 31+
+            (bowser.firefox && !bowser.android && bowser.version >= 31) ||
```

Comment:
> Also, would it be better to just deny known-bad stuff and let others give it a go?

I considered that, but we changed the wording such that they're allowed to try it and we're just warning them that it might not work, so I think an allowed list is better.